Client/src/components/contanier/Auth/index.js

```diff
@@ -1,27 +1,136 @@
-import React, { Component } from 'react'
+import React, {
+  Component
+} from 'react'
+import axios from 'axios';
 
 import Button from '../../common/Button'
 import LabeledInput from '../../common/LabeledInput'
 import logo from '../../assets/Image/amideasblue.png'
-import { firstLable, secondLable } from './staticdata'
 import './index.css'
 
 export default class Login extends Component {
 
+  state = {
+    id: '',
+    password: '',
+    loginError: '',
+    role: '',
+  };
+
+  handleChange = ({
+    target: {
+      name,
+      value
+    }
+  }) => {
+    this.setState({
+      [name]: value.trim()
+    });
+  };
+
   handleClick = (e) => {
-    e.preventdefault()
+    e.preventDefault()
+    const {
+      id,
+      password
+    } = this.state;
+    if (!id || !password) {
+      this.setState({
+        loginError: 'All Fields Requier'
+      })
+    } else if (id.length === 0) {
+      this.setState({
+        loginError: 'ID At least Five Numbers'
```

Comment:
are u ensure that your error statement suitable with the condition? 
the condition ` id.length === 0` and the statement  `ID At least Five Numbers` I think there is a conflict between them.

Reply:
and write the error statement like: `ID at least five numbers`